docs/man/git-lfs-migrate.1.ronn

```diff
@@ -192,25 +192,25 @@ include by `--include-ref` and not reachable by `--exclude-ref`.
        /         \
   A---B------C    refs/heads/my-feature
    \          \
-    \          refs/heads/master
+    \          refs/heads/main
      \
-      refs/remotes/origin/master
+      refs/remotes/origin/main
 ```
 
 In the above configuration, the following commits are reachable by each ref:
 
 ```
-refs/heads/master:         C, B, A
+refs/heads/main  :         C, B, A
```

Comment:
Here the whitespace is:
```
main  : A
```
while below it's
```
main:   A
```

Was that intentional, or just an inadvertent difference?

Reply:
It was an inadvertent difference.  Fixed.